components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/internal/CaptchaDataHolder.java

```diff
@@ -121,6 +123,14 @@ public void setReCaptchaRequestWrapUrls(String reCaptchaRequestWrapUrls) {
         this.reCaptchaRequestWrapUrls = reCaptchaRequestWrapUrls;
     }
 
+    public double getReCaptchaScoreThreshold() {
+        return reCaptchaScoreThreshold;
```

Comment:
formatting

Reply:
Don't get any issue in CheckStyle

Reply:
After method signature we should add one empty line. Please check other places as well.